hawkular-inventory-api/src/main/java/org/hawkular/inventory/base/Util.java

```diff
@@ -87,6 +87,7 @@ private Util() {
 
                     return ret;
                 } catch (Throwable e) {
+                    Log.LOGGER.error("\n\n\nTRANSACTION failed: " + e.getMessage());
```

Comment:
Yeah, I had this here for my own purposes, but I guess it can be there also in production. What about the log lvl? warn / debug?